Ferlab-Ste-Justine/clin-portal-ui
# feat: CLIN-2039 add landing page for clin genetic

# FEAT CLIN-2039 New landing page

- closes CLIN-2039

## Description

[CLIN-2039]

Acceptance Criterias

## Validation


## Screenshot
### Before
<img width="1920" alt="Capture d’écran, le 2023-07-17 à 17 27 37" src="https://github.com/Ferlab-Ste-Justine/clin-portal-ui/assets/82821620/eabcabab-04a7-42d2-96e8-941ac3a2c4f8">

### After
<img width="1920" alt="Capture d’écran, le 2023-07-17 à 17 27 20" src="https://github.com/Ferlab-Ste-Justine/clin-portal-ui/assets/82821620/c2189665-a646-4d7d-9857-a6e1c5209c08">

## QA

Check landing page is served by the portal instead of keycloak
Check that the lang switch works
Check that login button redirect to keycloak login page
Check that we can login
Check that after logout we are on the landing page

## Mention

@kstonge @luclemo
